<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preguntas_Algoritmos</class>
 <widget class="QMainWindow" name="Preguntas_Algoritmos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>844</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>844</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 34, 41);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="option2RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nota">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="btnMostrarRespuestas_2">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Mostar Respuestas</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Respuestas.png</normaloff>:/images/images/Respuestas.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="BotonSiguiente_2">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Siguiente</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Siguiente.png</normaloff>:/images/images/Siguiente.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BotonAnterior_2">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Anterior</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Anterior.png</normaloff>:/images/images/Anterior.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="botonVolver_2">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Menu.png</normaloff>:/images/images/Menu.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="option1RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="option4RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iD">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="option3RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);</string>
    </property>
    <property name="text">
     <string>Quiz sobre Algoritmos de Ordenamiento</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
